src/components/screens/LunchSection.tsx

```diff
@@ -76,10 +76,16 @@ const parseAttributes = (input?: string) => {
 const LunchOffer = ({ name, price }: { name: string; price: string }) => (
   <View style={styles.container}>
     <View style={styles.nameContainer}>
-      <RegularText accessibilityLabel={`Mittagstisch (Gerichtname): ${name}`}>{name}</RegularText>
+      <RegularText
+        accessibilityLabel={`${texts.eventRecord.description} ${consts.a11yLabel.lunch}`}
+      >
+        {name}
+      </RegularText>
     </View>
     <View style={styles.priceContainer}>
-      <RegularText accessibilityLabel={`Preis (${name}): ${price}`}>{price}</RegularText>
+      <RegularText accessibilityLabel={`${consts.a11yLabel.praice} (${name}): ${price}`}>
```

Comment:
there is a typo on the key here `praice -> price`

Reply:
solved f90ab7f

Reply:
same here

Reply:
now solved f90ab7f